broadcast-message.php

```diff
@@ -27,14 +27,19 @@
 
 require_once('common.php');
 
-authentication_require_role('global-admin');
+if ($CONF['sendmail_only_global_admin'] == 'YES' ) {
```

Comment:
That's the theory, but you'll never know about practise ;-)  (if someone edited config.inc.php instead of creating a config.local.php, we might end up with a config.inc.php.rpmnew). I also agree with David that you should flip the logic - "disable_feature = yes" is terribly confusing, "enable_feature = no" is much better.

BTW: Instead of checking $CONF directly, please use `Config::bool('broadcast_domain_admins')`. I know we didn't migrate everything yet, but new code should use it ;-)